src/connectors/menu/connectMenu.js

```diff
@@ -250,6 +250,49 @@ export default function connectMenu(renderFn, unmountFn) {
 
         return nextState;
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`